src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
+@import './utils/animation';
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the zero values should not have units. There is no need to specify units for zero values in `margin` and `padding` properties.